hydra-plutus/src/Hydra/Contract/Commit.hs

```diff
@@ -66,16 +65,15 @@ serializeCommit (i, o) = do
 
 -- | Decode an on-chain 'SerializedTxOut' back into an off-chain 'TxOut'.
 -- NOTE: Depends on the 'Serialise' instance for Plutus' 'Data'.
-deserializeCommit :: Commit -> Maybe (OffChain.TxIn, OffChain.TxOut CtxUTxO)
-deserializeCommit Commit{input, preSerializedOutput} =
+deserializeCommit :: OffChain.NetworkId -> Commit -> Maybe (OffChain.TxIn, OffChain.TxOut CtxUTxO)
+deserializeCommit networkId Commit{input, preSerializedOutput} =
   case deserialiseOrFail . fromStrict $ fromBuiltin preSerializedOutput of
     Left{} -> Nothing
     Right dat -> do
       txOut <- fromPlutusTxOut network <$> fromData dat
       pure (fromPlutusTxOutRef input, txOut)
  where
-  -- FIXME: not hard-code this
-  network = Testnet
+  network = networkIdToNetwork networkId
```

Comment:
Instead of this conversion here, I would expect `fromPlutusTxOut` to take a `cardano-api` type as it is defined in `hydra-cardano-api`.

Reply:
I moved the conversion to `fromPlutusTxOut` and made this function accept the `NewtorkId` instead of `Network`